### ISSUE TITLE
Code _em_observation_covariance

### ISSUE THREAD

**Issue by hallo-mudda** [pykalman/pykalman#51]
Hi All,

I'm new to this and not an expert of Kalman Filter. While reading through the code of the standard Kalman Filter, I saw that in the EM section in function _em_observation_covariance the observation covariance R is computed with transition matrix A. However, in the reference Abbeel, Pieter. “Maximum Likelihood, EM" http://www.cs.berkeley.edu/~pabbeel/cs287-fa11/slides/Likelihood_EM_HMM_Kalman-v2.pdf on p.22 and in the description added to the code the observation covariance is computed with the observation matrix C. Is this correct anyway?

Thanks

Best,

**Comment by peterasujan** [comment]
Yes, I just noticed the same thing, and I believe the implementation is wrong. It should definitely be using the observation covariance.
